### PR TITLE
[Test] Updating `relay.feature` to run curl command to enable E2E Relay Test

### PR DESCRIPTION
**Pull request by Olshansk**
## Summary

- Adding `APPGATE_SERVER` to the Makefile
- Adding the ability to run a `curl` command in the `e2e-test framework`
- Particl update to `relay.feature` to start sending a curl command as part of the E2E test

Testing the new test with:

```bash
export APPGATE_SERVER=http://localhost:42069 POCKET_NODE=tcp://127.0.0.1:36657 && \
POKTROLLD_HOME=./localnet/poktrolld && \
go test -run TestFeatures/Relay_Namespace/App_can_send_relay_to_Supplier -v ./e2e/tests/... -tags=e2e
```

## Type of change

Select one or more:

- [x] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [ ] **Run all unit tests**: `make go_develop_and_test`
- [ ] **Verify Localnet manually**: See the instructions [here](TODO: add link to instructions)

## Sanity Checklist

- [ ] I have tested my changes using the available tooling
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, updated documentation and left TODOs throughout the codebase

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 POKTROLLD_HOME := ./localnet/poktrolld
 POCKET_NODE = tcp://127.0.0.1:36657 # The pocket rollup node (full node and sequencer in the localnet context)
+RELAYER_NODE = http://anvil:8547 # TODO_IN_THIS_COMMIT: UPDATE THIS URL
```
Comment (h5law) [inline Makefile:5]: Is this URL going to be the RelayerProxy listening URL ultimately? For my own understanding

File: e2e/tests/init_test.go [hunk 2]
```diff
@@ -242,8 +242,12 @@ func (s *suite) TheSessionForApplicationAndServiceContainsTheSupplier(appName st
 	s.Fatalf("session for app %s and service %s does not contain supplier %s", appName, serviceId, supplierName)
 }
 
-func (s *suite) TheApplicationSendsTheSupplierARelayRequestForService(appName string, supplierName string, requestName string, serviceId string) {
-	// TODO(#126, @Olshansk): Implement this step
+func (s *suite) TheApplicationSendsTheSupplierARequestForServiceWithData(appName, supplierName, serviceId, requestData string) {
+	res, err := s.pocketd.RunCurl("", requestData)
```
Comment (h5law) [inline e2e/tests/init_test.go:246]: We want to make sure we append `/{service}` to the end of the rpcURL when we get there

File: e2e/tests/node.go [hunk 3]
```diff
@@ -95,3 +106,31 @@ func (p *pocketdBin) runCmd(args ...string) (*commandResult, error) {
 
 	return r, err
 }
+
+// runCurlPostCmd is a helper to run a command using the local pocketd binary with the flags provided
+func (p *pocketdBin) runCurlPostCmd(rpcUrl string, data string, args ...string) (*commandResult, error) {
```
Comment (h5law) [inline e2e/tests/node.go:111]:
```suggestion
func (p *pocketdBin) runCurlPostCmd(rpcUrlWithService string, data string, args ...string) (*commandResult, error) {
```

**Review by bryanchriswhite — APPROVED** [review]
LGTM! I don't see anything that's not already included in #178.

**Pull request merged**